packages/app-store-cli/src/build.ts

```diff
@@ -70,22 +71,30 @@ function generateFiles() {
     }
   });
 
-  function forEachAppDir(callback: (arg: App) => void) {
+  function forEachAppDir(callback: (arg: App) => void, filter: (arg: App) => boolean = () => true) {
```

Comment:
`filter` support to allow us to do things for apps statisfying a certain condition